starter_scripts/exaslct_within_docker_container.sh

```diff
@@ -11,12 +16,95 @@ else
   terminal_parameter=""
 fi
 
+declare -A relevant_mount_point_arguments
+relevant_mount_point_arguments["--flavor-path"]=in_path
+relevant_mount_point_arguments["--export-path"]=out_path
+relevant_mount_point_arguments["--output-directory"]=out_path
+relevant_mount_point_arguments["--temporary-base-directory"]=out_path
+relevant_mount_point_arguments["--cache-directory"]=in_path
+relevant_mount_point_arguments["--save-directory"]=out_path
+relevant_mount_point_arguments["--task-dependencies-dot-file"]=out_file
+relevant_mount_point_arguments["--test-folder"]=in_path
+relevant_mount_point_arguments["--test-file"]=in_file
+
+function _get_mount_point_docker_arg() {
+  mount_point_path=
+
+}
+function _get_mount_point_argument() {
+  current_arg=$1
+  next_arg=$2
+  arg_type=$3
+  #  temp="${next_arg%\"}" NEED TO CHECK IF I NEED ARGUMENTS WITH/WITHOUT QUOTES
+  #  temp="${temp#\"}"
+  case $arg_type in
+  in_path)
+    if [[ -d $next_arg ]]; then
+      local dir_name=$(readlink -f $next_arg)
+      mount_point_arguments+="-v $dir_name:$dir_name:ro " #MOUNT READONLY!!!
+    else
+      echo "Input directory $next_arg for parameter $current_arg does not exist."
+      exit 1
+    fi
+    ;;
+  in_file)
+    if [[ -f $next_arg ]]; then
+      local rel_dir_name="$(dirname "${next_arg}")"
+      local dir_name=$(readlink -f rel_dir_name)
+      mount_point_arguments+="-v $dir_name:$dir_name:ro " #MOUNT READONLY!!!
+    else
+      echo "Input file $next_arg for parameter $current_arg does not exist."
+      exit 1
+    fi
+    ;;
+  out_path)
+    #Create out directories if necessary
+    if [[ ! -d $next_arg ]]; then
+      mkdir -p "$next_arg"
+    fi
+    local dir_name=$(readlink -f $next_arg)
+    mount_point_arguments+="-v $dir_name:$dir_name "
+    ;;
+  out_file)
+    local rel_dir_name="$(dirname "${next_arg}")"
+    #Create out directories if necessary
+    if [[ ! -d $next_arg ]]; then
+      mkdir -p "$rel_dir_name"
+    fi
+    local dir_name=$(readlink -f rel_dir_name)
+    mount_point_arguments+="-v $dir_name:$dir_name "
```

Comment:
In the container, we need the original path and not the resolved path from `readlink -f`.